File: rpg-client-java/src/main/Nivel1.java
package main;

import static niveles.Nivel1.*;

/*
Crea tres variables de instancia: 
  - manzanas con un valor 10
  - fresas con valor a 5
  - frutas como suma de las dos variables anteriores
*/

public class Nivel1 {
	
	private int manzanas;
	private int fresas;
	private int frutas;
	

	public static void main(String[] args) {
		new Nivel1().comienzo();
	}
	
	public void comienzo() {

		iniciar();
		
		manzanas = 10;
		fresas = 5;
		frutas = fresas + manzanas;
		
		finalizar(this);
	}

}

File: rpg-client-javascript/src/nivel1.js
const {
  iniciar,
  finalizar
} = require('./niveles/nivel1');

iniciar();
/*
  Crea tres variables: 
    - manzanas con un valor 10
    - fresas con valor a 5
    - frutas como suma de las dos variables anteriores
*/

const manzanas = 10;
const fresas = 5;
const frutas = fresas + manzanas;

/* */
console.log({manzanas, fresas, frutas});


